src/SFA.DAS.Payments.AcceptanceTests/StepDefinitions/EarningAndPaymentsSteps.cs

```diff
@@ -574,20 +501,221 @@ public void ThenAPaymentIsDue(decimal dueAmount)
             //Get the due amount 
             var earning = PaymentsDueDataHelper.GetPaymentsDueForPeriod(EarningAndPaymentsContext.GetDefaultProvider().Ukprn,
                                                                         2016,
-                                                                        08,
+                                                                        09,
                                                                         environmentVariables)
                                                                         .FirstOrDefault();
 
             if (dueAmount != 0)
             {
                 Assert.IsNotNull(earning, $"Expected earning for the period but nothing found");
-                Assert.AreEqual(dueAmount, earning.AmountDue, $"Expected earning of {dueAmount} for period R02 but found {earning.AmountDue}");
+                Assert.AreEqual(dueAmount, earning.AmountDue, $"Expected earning of {dueAmount} for period R01 but found {earning.AmountDue}");
             }
             else
             {
                 Assert.IsNull(earning, $"There was no expected earning for the period but earnigs data found");
 
             }
         }
+
+        #region Payment Type Breakdown 

```

Comment:
The steps in this region should probably be put in a new step definitions class.